packages/hashes.csv

```diff
@@ -21,19 +21,19 @@ fetchai/agents/weather_station,QmaRVcpYHcyUR6nA1Y5J7zvaYanPr3jTqVtkCjUB4r9axp
 fetchai/connections/gym,QmXpTer28dVvxeXqsXzaBqX551QToh9w5KJC2oXcStpKJG
```

Comment:
for some reason many hashes were out-to-date, the PR doesn't change many of those packages.

Reply:
concerning

Reply:
are you on the right branch?

Reply:
I remind you we don't check hashes anymore.

Reply:
we do check hashes

Reply:
its concerning that you have locally differrent outcomes than me

Reply:
not for all packages:
https://github.com/fetchai/agents-aea/blob/develop/scripts/generate_ipfs_hashes.py#L445-L451